e2e/playwright/support/utils.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import type {Page} from '@playwright/test';
+
+import {UserProfile} from '@mattermost/types/users';
+import Client4 from '@mattermost/client/client4';
+
+export const waitForNewMessages = async (page: Page) => {
+    await page.waitForTimeout(1000);
+
+    // This should be able to be waited based on locators instead of pure time-based
+    // The following code work "almost" always. Commented for now to have green tests.
+    // await page.locator('#postListContent').getByTestId('NotificationSeparator').getByText('New Messages').waitFor();
+};
+
+export const getTodoBotDMPageURL = async (client: Client4, teamName: string, userId: string) => {
+    let team = teamName;
+    if (team === '') {
+        const teams = await client.getTeamsForUser(userId);
+        team = teams[0].name;
+    }
+    return `${team}/messages/@todo`;
+};
```

Comment:
This is the only thing specific to the todo plugin in this file. I think we have the function require a `botUsername` parameter so that we can pull this file out into shared code